code-generator.js

```diff
@@ -119,6 +139,13 @@ class PHPCodeGenerator {
             fullPath = basePath + '/' + elem.name + '.php';
             codeWriter = new codegen.CodeWriter(this.getIndentString(options));
             codeWriter.writeLine("<?php");
+
+            if (options.useStrictTypes === true) {
+                codeWriter.writeLine("");
```

Comment:
It was repeated code before, but with adding more I would either try to get the repeated code into a function or check if it is either a UMLClass, UMLInterface or UMLEnumeration and then output the common header. That would be an improvement, but functionally everything looks great 👍

Reply:
As soon as I have some time, I make some improvements to adjust this. Glad it was useful. :+1: